src/app/App.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import "./App.css";
+import { Counter, SearchForm, SelectTabs } from "../shared/ui";
+import { genresList } from "../shared/ui/SelectTabs/selectTabsMock";
+
+const App = () => {
+  const [selectedGenre, setSelectedGenre] = useState("ALL");
+
+  const handleSearch = (event) => {
+    event.preventDefault();
+  };
+
+  const handleChangeSelectedGenre = (event) => {
+    setSelectedGenre(event.target.value);
+  };
+  return (
+    <div className="main-page">
+      <Counter initialValue={5} />
+      <SearchForm
+        initialValue={""}
+        onSearch={handleSearch}
+        placeholder={"What do you want to search?"}
+      />
+      <SelectTabs
+        genres={genresList}
+        selectedGenre={selectedGenre}
+        onSelect={handleChangeSelectedGenre}
+      />
+    </div>
+  );
+};
+
+export default App;
```

Comment:
I didn't notice this default export, we should avoid it because https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a

e.g. `import Angular from './app/App'` 
I can write and would be okay, it can be a problem for us